features/doctrine/search_filter.feature

```diff
@@ -389,8 +462,7 @@ Feature: Search filter on collections
         "@id": {"pattern": "^/dummies$"},
         "@type": {"pattern": "^hydra:Collection$"},
         "hydra:member": {
-          "type": "array",
-          "maxItems": 0
```

Comment:
This is because the test was wrong in the first place, an invalid filter should not be applied and therefore should return every items of the collection. We fixed that here.